netmanager/src/views/pages/Reports/graph.js

```diff
@@ -69,12 +75,14 @@ const Graphs = props => {
   const [filterLocations,setFilterLocations] = useState([]);
 
   useEffect(() => {
-    fetch(constants.GET_MONITORING_SITES_LOCATIONS_URI)
-      .then(res => res.json())
-      .then((filterLocationsData) => {
-        setFilterLocations(filterLocationsData.airquality_monitoring_sites)
-      })
-      .catch(console.log)
+    // createAxiosInstance().get(constants.GET_MONITORING_SITES_LOCATIONS_URI, {
+    //   params: { token: accessToken }
```

Comment:
Hi @OchiengPaul442 , could we minimise or remove commented out code? Unless we are just temporarily disabling a feature?